src/main/java/com/openlattice/data/DataIntegrationApi.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018. OpenLattice, Inc.
+ * Copyright (C) 2020. OpenLattice, Inc.
```

Comment:
can we for sure change the copyright year?

Reply:
Not sure about that.. @geekbeast ?

Reply:
Thanks!